cdc/sinkv2/txneventsink/txn_event_sink.go

```diff
@@ -36,4 +39,35 @@ type TxnEventSink interface {
 	WriteTxnEvents(txns ...*TxnEvent) error
 	// Close closes the sink.
 	Close() error
+	// It's possible to dispatch tasks into several workers.
+	WorkerCount() int
+}
+
+// Worker is associated with a goroutine.
+type Worker interface{}
+
+type txnEventSinkImpl struct {
+	input   chan<- *TxnEvent
+	workers []Worker
+}
+
+func NewTxnEventSinkImpl() TxnEventSink {
+	sink := &txnEventSinkImpl{
+		input:   make(chan<- *TxnEvent, math.MaxUint64), // Unbounded channel.
```

Comment:
I think this can actually result in asking the operating system for `math.MaxUint64` bytes of memory.

Reply:
Just a quick show. Should use a real unbounded channel instead.

Reply:
okay, cool